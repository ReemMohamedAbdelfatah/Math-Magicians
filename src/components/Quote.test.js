import React from 'react';
import { render } from '@testing-library/react';
import Quotes from './Quote';

describe('Quotes', () => {
  it('Quotes to match Snapshot', () => {
    const quotes = render(<Quotes />);
    expect(quotes).toMatchSnapshot();
  });
});
