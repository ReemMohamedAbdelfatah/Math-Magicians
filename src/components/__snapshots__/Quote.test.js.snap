// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Quotes component Test component should render properly 1`] = `
<div
  className="quotes"
>
  <p
    className="author"
  >
    Loading quotes...
  </p>
</div>
`;
